### DISCUSSION TITLE
Issue when building (using 9 instructions on 8 )

### DISCUSSION THREAD

**Discussion by NicTanghe** [dreamworksanimation/openmoonray#118] [Q&A]
Hello when trying to build on rhel-8 I get the following error, but I’m a bit confused on how that’s possible as far as I understand it he should have just build it himself, so why didn’t he find it ?

Any help solving this is greatly appreciated.

```
Consolidate compiler generated dependencies of target PyOpenColorIO
/usr/bin/ld: CMakeFiles/ociolutimage.dir/main.cpp.o: undefined reference to symbol '_ZN16OpenImageIO_v2_310ParamValue11clear_valueEv'
//usr/local/lib64/libOpenImageIO_Util.so.2.3: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
```

**Comment by shdwdln** [comment]
Yeah I am getting this error too. Want to bump this.

**Comment by Shantarli** [comment]
I also faced this problem. I assume this is due to the fact that the libraries were already compiled in previous attempts. 
I just deleted everything OpenImage related in usr/local/lib64 and was able to compile the dependencies again without any problems.
